main.go

```diff
@@ -1,25 +1,50 @@
 package main
 
 import (
-	"log"
 	"os"
 
+	"github.com/alecthomas/kingpin/v2"
 	"github.com/app-sre/acs-cve-prom-exporter/cmd/exporter"
+	"github.com/go-kit/log/level"
+	"github.com/prometheus/common/promlog"
+	"github.com/prometheus/common/promlog/flag"
+	"github.com/prometheus/common/version"
 )
 
 const (
 	AcsEndpoint = "ACS_ENDPOINT"
 	AcsToken    = "ACS_TOKEN"
 )
 
+var (
+	listenAddress = kingpin.Flag("web.listen-address", "The address to listen on for HTTP requests.").Default(":9090").String()
+	metricsPath   = kingpin.Flag("web.telemetry-path", "Path under which to expose metrics.").Default("/metrics").String()
+	namespace     = "acs_cve_prom_exporter"
+	shortFlagName = 'h'
+)
+
 func main() {
+	promLogConfig := &promlog.Config{}
+	flag.AddFlags(kingpin.CommandLine, promLogConfig)
+	kingpin.Version(version.Print(namespace))
+	kingpin.HelpFlag.Short(shortFlagName)
+	kingpin.Parse()
+	logger := promlog.New(promLogConfig)
 	endpoint := os.Getenv(AcsEndpoint)
 	if endpoint == "" {
-		log.Fatalf("Required var missing: %s", AcsEndpoint)
+		level.Error(logger).Log("msg", "Required variable missing", AcsEndpoint)
+		os.Exit(1)
 	}
 	token := os.Getenv(AcsToken)
 	if token == "" {
-		log.Fatalf("Required var missing: %s", AcsToken)
+		level.Error(logger).Log("msg", "Required variable missing", AcsEndpoint)
```

Comment:
`AcsEndpoint` -> `AcsToken`

Reply:
https://github.com/app-sre/acs-cve-prom-exporter/pull/13/commits/19c394fd8c95662e181405bcb99a9b45a41f284d

Reply:
I think you mean to output `ACS_TOKEN` (`AcsToken` value), `token` is empty string in the error case